Makefile

```diff
@@ -1,18 +1,20 @@
 TARGET	=	tiramisu
-SRC	:=	src/tiramisu.c src/events.c src/notification.c
```

Comment:
evolution-mail-notification
evolution
0
New email in Evolution
You have received 4 new messages.
desktop-entry=org.gnome.Evolution|urgency=1
Show INBOX=default
-1